src/client/sandbox/event/listening-context.ts

```diff
@@ -29,6 +29,7 @@ export function addListeningElement (el, events) {
                 internalHandlers:     [],
                 outerHandlers:        [],
                 outerHandlersWrapper: null,
+                postHandlers:         [],
```

Comment:
I don't like this name because it can be understood in several ways.

It's internal hammerhead handlers, which are called last.

I guess, the `internalHandlers` are handlers which are called before users' scripts.
So, I propose to clean up the code and perform some renames:
`internalHandlers -> internalBeforeHanlders`
`postHandlers -> internlaAfterHanlders`

@LavrovArtem What do you think about it?

Reply:
I like this suggestion